c10/util/complex.h

```diff
@@ -515,14 +515,7 @@ namespace c10_internal {
 template <typename T>
 C10_HOST_DEVICE constexpr thrust::complex<T>
 cuda101bug_cast_c10_complex_to_thrust_complex(const c10::complex<T>& x) {
-#if defined(CUDA_VERSION) && (CUDA_VERSION < 10020)
```

Comment:
similarly here, `cuda101bug...` uses should just be replaced with  static_cast, as the comment suggests